TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -226,7 +223,7 @@ public static OptionSet GetCommonTypeCobolOptions(TypeCobolConfiguration typeCob
                 { "i|input=", "{PATH} to an input file to parse. This option can be specified more than once.", v => typeCobolConfig.InputFiles.Add(v) },
                 { "o|output=","{PATH} to an output file where to generate code. This option can be specified more than once.", v => typeCobolConfig.OutputFiles.Add(v) },
                 { "d|diagnostics=", "{PATH} to the error diagnostics file.", v => typeCobolConfig.ErrorFile = v },
-                { "s|skeletons=", "{PATH} to the skeletons file.", v => typeCobolConfig.skeletonPath = null },
+                { "s|skeletons=", "{PATH} to the skeletons file. DEPRECATED : generation using dynamic skeleton is not supported.", v => {}},
```

Comment:
Maybe the following mesage is better "no longer"

DEPRECATED : generation using dynamic skeleton is no longer supported.

Reply:
Yes. Done in e411a2e3ed3760b9424a3791b08505e9617d8723.